atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/TableSplittingKeyValueService.java

```diff
@@ -89,6 +92,13 @@ public void createTable(String tableName, byte[] tableMetadata) {
 
     @Override
     public void createTables(Map<String, byte[]> tableNamesToTableMetadata) {
+        Map<KeyValueService, Map<String, byte[]>> splitTableNamesToTableMetadata = groupByDelegate(tableNamesToTableMetadata);
+        for (KeyValueService delegate : splitTableNamesToTableMetadata.keySet()) {
```

Comment:
this case doesn't matter for perf at all and you don't have to change it, but just for SA at least my IDE would yell at me if I wrote this, because using the entrySet here instead is more efficient since you're also doing the get(key) inside here